packages/clay-form/CHECKBOX.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Checkbox'
 description: 'A checkbox is a component that lets the user select the value that is written in its corresponding text label. A user can select multiple checkboxes from a group at the same time.'
 lexiconDefinition: 'https://liferay.design/lexicon/core-components/forms/radio-check-toggle/'
 packageNpm: '@clayui/form'
-navigationParent: '/docs/components/form.html'
+navigationParent: 'docs/components/form.html'
```

Comment:
I removed the leading `/` so that we don't have to manually add that when comparing paths

Reply:
Yea this is smarter than manually adding it when comparing paths. 👍